src/main/java/com/algorand/algosdk/v2/client/model/HealthCheck.java

```diff
@@ -11,7 +12,7 @@
 public class HealthCheck extends PathResponse {
```

Comment:
FYI, I noticed this when working on the response generator and didn't realize it was happening here as well. This isn't very important, but figured I should mention it since it would be nice to use `*Response` names everywhere eventually.

"Aliases" are not handled currently. We have `HealthCheckResponse` which references `HealthCheck`

I have some code in a branch that I should PR which adds a new `StructDef` field called `aliasOf`, and adds a new event like `StructDef(name="HealthCheckResopnse", aliasOf="HealthCheck")`

```
{
   "responses": {
      "HealthCheckResponse": {
         "description": "(empty)",
         "schema": {
            "$ref": "#/definitions/HealthCheck"
         }
      }
   }
   "definitions": {
     "HealthCheck": {
       "description": "A health check response.",
        "type": "object",
        "required": [
           "message"
        ],
        "properties": {
           "data": {
              "type": "object"
           },
           "message": {
              "type": "string"
           }
        }
     }
  }
}
```

Reply:
Also I'm not sure why I gave HealthCheck a data object, that seems like a mistake in the spec.

Reply:
This is something we can think about in the future. Now, v2 is released and making changes won't go well I guess.